homestar-runtime/src/logger.rs

```diff
@@ -43,31 +43,60 @@ fn init(
     guard: WorkerGuard,
     #[allow(unused_variables)] settings: &settings::Monitoring,
 ) -> WorkerGuard {
+    // RUST_LOG ignored when EVERY_CLI is true
+    let every_cli = std::env::var("EVERY_CLI").is_ok_and(|val| val == "true");
+
     // TODO: Add support for customizing logger(s) / specialzed formatters.
-    let format_layer = tracing_logfmt::builder()
-        .with_level(true)
-        .with_target(true)
-        .with_span_name(true)
-        .with_span_path(true)
-        .with_location(true)
-        .with_module_path(true)
-        .layer()
-        .with_writer(writer);
+    let format_layer = if every_cli {
+        tracing_logfmt::builder()
+            .with_level(true)
+            .with_target(false)
+            .with_span_name(false)
+            .with_span_path(false)
+            .with_location(false)
+            .with_module_path(false)
+            .layer()
+            .with_writer(writer)
+    } else {
+        tracing_logfmt::builder()
+            .with_level(true)
+            .with_target(true)
+            .with_span_name(true)
+            .with_span_path(true)
+            .with_location(true)
+            .with_module_path(true)
+            .layer()
+            .with_writer(writer)
+    };
 
-    let filter = EnvFilter::try_from_default_env().unwrap_or_else(|_| {
-        EnvFilter::new("info")
-            .add_directive("homestar_wasm=info".parse().expect(DIRECTIVE_EXPECT))
-            .add_directive("libp2p=info".parse().expect(DIRECTIVE_EXPECT))
+    let filter = if every_cli {
+        EnvFilter::new("off")
```

Comment:
My only possible qualm here is we won't get general errors (we can skip all deps ones) that could be helpful at the highest level (remembering that `error`) is the highest level. Nonetheless, I won't be a stickler here, but maybe worth chatting with @avivash?